Tests/DVRTests/SessionTests.swift

```diff
@@ -108,6 +108,30 @@ class SessionTests: XCTestCase {
 
         waitForExpectations(timeout: 1, handler: nil)
     }
+    
+    func testTextPlaybackWithParams() {
```

Comment:
you can send 2 requests, one with a key another one without a key. And they should succeed and provide the same response

Reply:
Done